src/plotting/plotSettings.component.tsx

```diff
@@ -360,7 +360,7 @@ const PlotSettings = (props: PlotSettingsProps) => {
                     label="Linear"
                   />
                   <FormControlLabel
-                    value="log"
+                    value="logarithmic"
```

Comment:
If this is changing, then we should also update the AxisSettings interfaces to say 'logarithmic' instead of just 'log' in `app.types.tsx`. Frankly, I'm not even sure how this is working considering `changeXAxisSettings` should only be accepting a value of time, linear or log. Make sure to do this for both `XAxisSettings` and `YAxisSettings`

Reply:
I thought I had changed the value in the types, obviously I'm misremembering. Good catch - it should definitely be changed in the types.

It's not caught as `value` is cast to `XAxisSettings['scale']` as otherwise it's just a `string`, and Typescript only catches build time errors not runtime

Reply:
Ohhh that makes sense, I wouldn't have thought of that. Yes, if there's a way to import the ChartJS scales and remove any of our own defined ones, I think we should definitely do this. Ensures that everything is correct and improves maintainability

Reply:
I was more meaning to type the chart.js options objects, but I can see if we can do that as well. I'll ensure that I add chart.js packages as dev dependencies only